<html>
<head>
	<title> Avanti Shrikumar </title>
	<link rel="stylesheet" href="default.css" type="text/css">
	<meta name="keywords" content="Avanti Shrikumar, Av Shrikumar">
</head>

<body>
	<link href="default.css" rel="stylesheet" type="text/css">
	
	<div class="navigation">
		<a class="present" href="index.html"> Home </a>
		<a class = "menu" href="resume.html"> Resume </a>
		<a class="menu" href="projects.html"> Fun </a>
		<a class="menu" href="bio.html"> Bio </a>
	</div>
	
<div class="content">	
	<h1> Avanti Shrikumar</h1>	
	<p>
		In 2020-2022, I was a Ram and Vijay Shriram postdoctoral fellow in the Department of Earth System Science at Stanford, where I worked with Professor Karen Casciotti on computational methods for oceanographic data analysis. During my postdoc, I developed <a href="https://www.essoar.org/doi/abs/10.1002/essoar.10507053.4">pyOMPA</a>, an improved method for performing water mass mixing analysis. I received my PhD from the Department of Computer Science at Stanford, advised by Professor <a href="https://sites.google.com/site/anshulkundaje/">Anshul Kundaje</a>. During my PhD, I developed methods to make deep learning models interpretable, and applied these methods to study regulatory genomics. In order of my excitement, my PhD projects were:
	</p>
	<p>
	<img  src="headshot_AvantiShrikumar_lowerRes.png" id="me" align="right">	
		<ul>
			<li> <a href="https://github.com/kundajelab/tfmodisco">TF-MoDISco</a>, a method that leverages input-level importance scores produced by deep learning models or SVMs to reveal recurring patterns in genomic data.
			<li> <a href="https://github.com/kundajelab/deeplift">DeepLIFT</a>, a computationally efficient method for generating input-level importance scores to explain individual predictions made by a deep learning model.
			<li> <a href="https://academic.oup.com/bioinformatics/article/35/14/i173/5529147">GkmExplain</a>, a computationally efficient method for generating input-level importance scores to explain individual predictions made by nonlinear gapped-kmer support vector machines.
			<li> <a href="https://arxiv.org/abs/1901.06852">Adapting to label shift with Bias-Corrected Calibration</a>, where we show that performing domain adaptation to label shift using a combination of maximum likelihood estimation and specific types of bias-corrected calibration gives state-of-the-art results at domain adaptation to label shift.
			<li> <a href="https://arxiv.org/abs/1802.07024">A Flexible and Adaptive Framework for Abstention under Class Imbalance</a>: a general method for deciding which examples a predictive model should abstain on to optimize metrics other than standard classification accuracy (which is often not very useful when there is class imbalance).
			<li> <a href="https://icml-compbio.github.io/2020/papers/WCBICML2020_paper_58.pdf"> A novel strategy for determining when featured interactions learned by a neural network might be false positives</a>, based on looking at the impact that the learned interaction has on the model's prediction loss on held-out data</li>
			<li> <a href="https://www.biorxiv.org/content/early/2017/07/07/146431">Separable fully-connected layers</a>, a novel architectural component for deep learning models that takes advantage of recurring positional patterns in genomic data. 
			<li> <a href="https://www.biorxiv.org/content/early/2017/01/27/103663">Reverse-complement parameter sharing</a>, a novel architectural component for deep learning models that takes advantage of symmetries induced by the double-stranded nature of DNA.
			<li> <a href="https://github.com/kundajelab/simdna">simDNA</a>, a package for generating simulated regulatory genomic sequences.
		</ul>
	</p>
        <p>
		 I have a Bachelor's in Computer Science with Molecular Biology from MIT and spent a year working as a developer for the <a href="https://www.palantir.com/solutions/healthcare-delivery/"> Healthcare team of Palantir Technologies</a> before starting my PhD.
	</p>
	<p>
	  <img class="email" src="email.png" style="border: 0px;">
	</p>
</body>
</html>
