<html>
<head>
	<title> Avanti Shrikumar</title>
	<link rel="stylesheet" href="default.css" type="text/css">
	<meta name="keywords" content="Avanti Shrikumar, Av Shrikumar">
</head>

<body>
	<link href="default.css" rel="stylesheet" type="text/css">	
	<div class="navigation" >
	<a class="menu" href="index.html"> Home </a>
	<a class = "present" href="resume.html"> Resume </a>
	<a class="menu" href="projects.html"> Fun </a>
	<a class="menu" href="bio.html"> Bio </a>
	
	<h1>
		<a class="present" href="resume.html"> Resume  </a>    
		
	</h1>
		
	</div>
	<div class="content">	
		<!-- <h3><a href="http://web.stanford.edu/~avanti/resume.pdf">(PDF)</a><h3><--!>
		<h2><a name="education">Education</a></h2>
		<table>
			<tr>
				<td class ="resume-left">Stanford
		  	<br> 2014 - <i>(2019)</i>
		    <br> Ph.D.
				  	</td> 
				<td class="resume-right"> 
					Ph.D. in Computer Science, applying deep learning to study regulatory genomics
				    <br>Ph.D. Advisor: Anshul Kundaje.<br>
				</td>
			</tr>
			<tr>
				<td class="resume-left"> MIT
				<br>2009 - 2013
				<br> B.S.
				</td>
				<td class="resume-right">
					B.S. Computer Science with Molecular Biology, Minor in Mathematics
					<br>Undergraduate GPA: 5.0/5.0
				</td>
			</tr>
		</table>
		<h2><a name="experience">Experience</a></h2>
		<table>
			<tr>
		        <td class="resume-left">
		          <a href="https://sites.google.com/site/anshulkundaje/">Kundaje Lab</a>
		          <br>
		          Sep 2014 - Present
		        </td>
		        <td class="resume-right">
					<ul>
				        <li>Developed <a href="https://github.com/kundajelab/deeplift">DeepLIFT</a> for explaining the individual predictions made by a deep learning model</li>	
					<li>Developed <a href="https://github.com/kundajelab/tfmodisco">TF-MoDISco</a> for understanding the patterns learned by a deep learning model.</li>
					<li>Developed <a href="https://arxiv.org/abs/1802.07024">Curve Optimization</a>, a novel method for deciding which examples a predictive model should abstain on.</li>
					<li>Developed <a href="https://academic.oup.com/bioinformatics/article/35/14/i173/5529147">GkmExplain</a> for explaining the individual predictions made by nonlinear gapped-kmer Support Vector Machines.</li>
					<li>Developed <a href="https://www.biorxiv.org/content/early/2017/07/07/146431">Separable Fully-Connected Layers</a> and <a href="https://www.biorxiv.org/content/early/2017/01/27/103663">Reverse-Complement Parameter Sharing</a> to exploit unique properties of genomic data for deep learning.</li>
					<li>Developed an <a href="https://github.com/kundajelab/simdna">engine for creating simulated genomic datasets</a>; used in multiple projects to generate a simulated dataset to empirically validate a method.</li>
					<li>Developed an <a href="https://github.com/kundajelab/momma_dragonn">internal framework</a> to facilitate the training of deep learning models in the Kundaje lab.</li>
                                       </ul>
		        </td>
	      	</tr>
			<tr>
		        <td class="resume-left">
		          <a href="https://www.palantir.com/solutions/healthcare-delivery/">Palantir Technologies</a>
		          <br>
		          June 2013 - Sep 2014
		        </td>
		        <td class="resume-right">
		         Forward Deployed Engineer for the Healthcare Team.<ul><li>Primary developer for pilot project which successfully became the Healthcare Team's first enterprise deployment. Continued as primary developer until I left the company.</li><li>Created a way to integrate Palantir's two major platforms at the time into a seamless experience; success was highlighted company-wide.<li>Went beyond my assigned duties to build multiple handy software utilities for use team-wide and company-wide.</li><li>Consistently rated high on bi-annual reviews. Was told that I had at times done the work of 2-3 coders and had set a new standard for productivity among my teammates.</li></ul>
		        </td>
		      </tr>
		</table>
		<h2><a name="talks">Selected Talks</a></h2>
		<table>
			<tr>
				<td class="resume-left">
					<a href="https://meetings.cshl.edu/abstracts.aspx?meet=DATA&year=18">Biological Data Science</a><br> Nov 2018
				</td>
				<td class="resume-right">
					<em>Suggested best practices for interpreting deep learning models of regulatory DNA</em><br> Slides <a href="https://drive.google.com/open?id=1ROb9Jjinz2EsoEA-M671Z-ZxIDamBnyM">here</a>.
				</td>
			</tr>
			<tr>
				<td class="resume-left">
					<a href="https://2018gputechconf.smarteventscloud.com/connect/sessionDetail.ww?SESSION_ID=157363">NVIDIA GTC</a><br> March 2018
				</td>
				<td class="resume-right">
					<em>Not Just a Black Box: Interpretable Deep Learning for Genomics and Beyond</em><br> Video <a href="https://youtu.be/T8AMLwmy_vQ">here</a>.
				</td>
			</tr>
			<tr>
				<td class="resume-left">
					<a href="https://mlcb.github.io/">NIPS MLCB</a><br> Dec 2017
				</td>
				<td class="resume-right">
					<em>TF-MoDISco: Deep learning non-redundant, predictive sequence motifs of transcription factors</em><br> Video <a href="https://www.youtube.com/watch?v=fXPGVJg956E">here</a>.
				</td>
			</tr>
			<tr>
				<td class="resume-left">
					<a href="http://proceedings.mlr.press/v70/shrikumar17a">ICML 2017</a><br> Aug 2017
				</td>
				<td class="resume-right">
					<em>Learning Important Features Through Propagating Activation Differences</em><br> Video and slides <a href="https://vimeo.com/238275076">here</a> and <a href="https://docs.google.com/file/d/0B15F_QN41VQXSXRFMzgtS01UOU0/edit?usp=docslist_api&filetype=mspresentation">here</a>, paper <a href="https://arxiv.org/pdf/1704.02685">here</a>.
				</td>
			</tr>
			<tr>
				<td class="resume-left">
					<a href="https://www.re-work.co/events/deep-learning-health-boston-2017/schedule">Deep Learning In Healthcare Summit, Boston</a><br> May 2017
				</td>
				<td class="resume-right">
					<em>Not Just a Black Box: Interpretable Deep Learning for Genomics</em><br> Interview <a href="https://www.re-work.co/blog/deep-learning-genomics-avanti-shrikumar-stanford-university">here</a>.
				</td>
			</tr>
			<tr>
				<td class="resume-left">
					<a href="https://web.stanford.edu/dept/cehg/cgi-bin/cehg-symposium/speakers/avanti-shrikumar/">CEHG Symposium</a><br> March 2016
				</td>
				<td class="resume-right">
					<em>Not Just a Black Box: Interpretable Deep Learning for Genomics</em><br> Video <a href="https://www.youtube.com/watch?v=DRujWGpqAsQ">here</a>.
				</td>
			</tr>
		</table>
		<h2><a name="publications">Selected Publications</a></h2>
		<table>
			<tr>
				<td class="resume-left">Proceedings of the ISMB<br>July 2019</td>
				<td class="resume-right">
			    <a href="https://academic.oup.com/bioinformatics/article/35/14/i173/5529147">Gkmexplain: Fast and Accurate Interpretation of Nonlinear Gapped k-mer SVMs</a>.<br>
			    <em><u>Avanti Shrikumar*&dagger;</u>, Eva Prakash*</em>, <em>Anshul Kundaje</em>&dagger;.<br> *co-first authors <br> &dagger; co-corresponding authors<br><br> 
                                 Computationally efficient algorithm for explaining individual predictions made by nonlinear gapped-kmer SVMs trained on genomic sequences.<br><br></td>
			</tr>
			<tr>
				<td class="resume-left">Nature Biotechnology<br>May 2019</td>
				<td class="resume-right">
			    <a href="https://www.nature.com/articles/s41587-019-0140-0">Kipoi: accelerating the community exchange and reuse of predictive models for genomics</a>.<br>
			    <em>&#381;iga Avsec*</em>&dagger;<em>, Roman Kreuzhuber*,  Johnny Israeli, Nancy Xu, Jun Cheng,  <u>Avanti Shrikumar</u>, Abhimanyu Banerjee,  Daniel S. Kim,  Lara Urban, Anshul Kundaje</em>&dagger;<em>,  Oliver Stegle</em>&dagger;<em>, Julien Gagneur</em>&dagger;<br> *co-first authors <br> &dagger; co-corresponding authors <br><br> 
                                 Model zoo for genomics. I was involved in designing the API during the early stages and converted the DeepBind models to Keras using the code <a href="https://github.com/kundajelab/DeepBindToKeras">here</a><br><br></td>
			</tr>
			<tr>
				<td class="resume-left">arXiv<br>Jan 2019</td>
				<td class="resume-right">
			    <a href="https://arxiv.org/abs/1901.06852">Adapting to Label Shift with Bias-Corrected Calibration</a>.<br>
			    Amr Alexandari*,<em>Anshul Kundaje</em>&dagger;<em>,<u>Avanti Shrikumar*&dagger;</u></em>.<br> *co-first authors <br> &dagger; co-corresponding authors<br><br>
				 Studied the impact of calibration on domain adaptation to label shift in the context of modern neural networks. Identified a principled strategy for computing source-domain priors in EM-based domain adaptation that is particularly important when the calibrated probabilities have systematic bias. When source-domain priors are computed using our proposed approach, label shift estimates from EM can perform surprisingly well compared to the recently-proposed <a href="https://arxiv.org/abs/1802.03916">Black Box Shift Estimation (ICML 2018)</a> even when predictions are not calibrated. In experiments with image classification & diabetic retinopathy detection, found that the best results were obtained by using EM-based domain adaptation with a calibration approach that contains class-specific bias parameters capable of reducing systematic bias.<br><br>
				</td>
			</tr>
			<tr>
				<td class="resume-left">arXiv<br>July 2018</td>
				<td class="resume-right">
			    <a href="https://arxiv.org/abs/1802.07024">Computationally Efficient Measures of Internal Neuron Importance</a>.<br>
			    <em><u>Avanti Shrikumar*&dagger;</u>, Jocelin Su*</em>, <em>Anshul Kundaje</em>&dagger;.<br> *co-first authors <br> &dagger; co-corresponding authors<br><br>
				 Showed an equivalence between <a href="https://arxiv.org/abs/1805.12233v1">Total Conductance</a>, a recently-proposed method for computing internal neuron importance, and <a href="https://arxiv.org/abs/1703.01365">Path Intergrated Gradients</a>, thereby providing a computationally efficient way to compute the former. The reformulation of Total Conductance was referred to as Neuron Integrated Gradients. Benchmarked Neuron Integrated Gradients against DeepLIFT. Colab notebook reproducing results <a href="http://bit.ly/neuronintegratedgradients">here</a><br><br>
				</td>
			</tr>
		    	<tr>
				<td class="resume-left">
			        Journal of the Royal Society Interface <br> April 2018
				</td>
				<td class="resume-right">
			    <a href="https://royalsocietypublishing.org/doi/10.1098/rsif.2017.0387">Opportunities And Obstacles For Deep Learning In Biology And Medicine</a><br><br>
						Collaboratively written review on deep learning for biology and medicine. I wrote the <a href="https://github.com/greenelab/deep-review/blob/71bb07775266206a497c013796e2c6c2258b8a78/sections/06_discussion.md#interpretation">section on interpretation.</a> Note that the final published version was stripped down due to word limits. I have linked to the original submission from my end.<br><br>
				</td>
		    	</tr>
			<tr>
				<td class="resume-left">arXiv<br>Feb 2018</td>
				<td class="resume-right">
			    <a href="https://arxiv.org/abs/1802.07024">A Flexible and Adaptive Framework for Abstention Under Class Imbalance</a>.<br>
			    <em><u>Avanti Shrikumar*</em>&dagger;</u>,Amr Alexandari*, <em>Anshul Kundaje</em>&dagger;.<br> *co-first authors <br> &dagger; co-corresponding authors<br><br>
				 Proposes a framework for identifying which examples to abstain on in order to optimize for a specific metric of interest. Leverages the insight that because the calibrated probabilities can be used as a proxy for the true label, optimization is possible even when the ground-truth labels are not known. Derived computationally efficient algorithms for optimizing auROC, sensitivity at a target specificity and weighted cohen's kappa. Showed that by leveraging strategies for domain adaptation to label shift, the abstention algorithms can apply even in situations where the test-set distribution has a different class imbalance compared to the training-set distribution.<br><br>
				</td>
			</tr>
			<tr>
				<td class="resume-left">
			        ICML Comp Bio Workshop (Spotlight Talk)<br> June 2017
				</td>
				<td class="resume-right">
			    <a href="https://www.biorxiv.org/content/early/2017/06/10/146431">Separable Fully-Connected Layers Improve Deep Learning Models For Genomics</a>.<br>
			    <em>Amr Alexandari*, <u>Avanti Shrikumar</u>*, Anshul Kundaje</em>. <br>*co-first authors<br><br>
						Adapts deep learning models for genomics by leveraging known patterns in transcription factor binding data.<br><br>
			</td>
		    </tr>
			<tr>
				<td class="resume-left">
			        ICML <br> April 2017
				</td>
				<td class="resume-right">
			    <a href="https://arxiv.org/abs/1704.02685">Learning Important Features Through Propagating Activation Differences</a>.<br>
			    <em><u>Avanti Shrikumar</u>, Peyton Greenside, Anshul Kundaje</em>.<br><br>
			Details a computationally efficient algorithm to explain individual predictions of a deep learning model by assigning contribution scores to individual parts of the input. Code <a href="https://github.com/kundajelab/deeplift">here</a>.<br><br>
			</td>
		    </tr>
			<tr>
				<td class="resume-left">
			        BioRxiv <br> January 2017
				</td>
				<td class="resume-right">
			    <a href="https://www.biorxiv.org/content/early/2017/01/27/103663">Reverse-Complement Parameter Sharing Improves Deep Learning Models For Genomics</a>.<br>
			    <em><u>Avanti Shrikumar</u>*, Peyton Greenside*, Anshul Kundaje</em>.<br>*co-first authors<br><br>
						Adapts deep learning models for genomics by leveraging the reverse-complement property of DNA sequence.<br><br>
			</td>
		    </tr>
			<tr>
				<td class="resume-left">
				Circulation Research
				<br>Dec 2014
				</td>
				<td class="resume-right">
					<a href="http://www.ncbi.nlm.nih.gov/pubmed/25477501">Transcriptional Reversion of Cardiac Myocyte Fate During Mammalian Cardiac Regeneration</a>.<br>
                                        <em>O'Meara CC, Wamstad JA, Gladstone RA, Fomovsky GM, Butty VL, <u>Shrikumar A</u>, Gannon JB, Boyer LA, Lee RT</em><br><br>

					Collaboration between Boyer lab at MIT and Lee lab at Harvard. I analysed RNA-seq data to study transcriptional reversion. <br><br>
				</td>
			</tr>
			<tr>
				<td class="resume-left">
				Cell <br> Sep 2012
				</td>
				<td class="resume-right">
                                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/22981692">Dynamic and coordinated epigenetic regulation of developmental transitions in the cardiac lineage</a>.<br>
                                        <em>Wamstad JA, Alexander JM, Truty RM, <u>Shrikumar A</u>, Li F, Eilertson KE, Ding H, Wylie JN, Pico AR, Capra JA, Erwin G, Kattman SJ, Keller GM, Srivastava D, Levine SS, Pollard KS, Holloway AK, Boyer LA, Bruneau BG.</em><br><br>
					Collaboration between Boyer lab at MIT and Gladstone Institutes. I performed the bulk of bioinformatics analysis at the Boyer lab. 355 citations as of Jan 2018.<br><br>
				</td>
			</tr>
		</table>
		<h2><a name="recognition">Recognition</a></h2>
		<table>
			<tr>
			  <td class="resume-left">HHMI International Student Research Fellowship<br>2016</td>
			  <td class="resume-right">Awarded to 20 international students. Announcement <a href="https://www.hhmi.org/news/hhmi-selects-20-international-student-research-fellows">here</a>.</td>
			</tr>
			<tr>
			  <td class="resume-left">Stanford Bio-X Fellowship<br>2016</td>
			  <td class="resume-right">Bio-X fellowships are awarded to about 25 students annually for interdisciplinary research. Announcement <a href="https://biox.stanford.edu/about/people/phd-fellows/avanti-shrikumar-bio-x-bowes-fellow">here</a>.</td>
			</tr>
			<tr>
			  <td class="resume-left">Microsoft Women's Fellowship<br>2016</td>
			  <td class="resume-right">Awarded to one woman per participating University pursuing or interested in pursuing a PhD. Announcement <a href="https://www.microsoft.com/en-us/research/blog/microsoft-fellowship-program-supports-women-computing/">here</a>.</td>
			</tr>
			<tr>
			  <td class="resume-left"> Outstanding Research Award<br> Spring 2013</td>
			  <td class="resume-right">Awarded to 3 projects completed as part of MIT's SuperUROP program. My project was done in the Kellis lab. Announcement <a href="http://web.mit.edu/annualreports/pres13/2013.04.05.pdf">here</a>.</td>
			</tr>
			<tr>
			  <td class="resume-left"> Sophomore Academic Excellence Award<br> Fall 2011</td>
			  <td class="resume-right">AIChE Sophomore Academic Excellence Award for the student with the highest GPA among chemical engineers after sophomore year at MIT. Announcement <a href="http://fscarbonmanagement.org/sites/default/files/chenected/2011/10/6680-AIChEStudentAwardsCrmy.pdf">here</a>.</td>
			</tr>
			<tr>
			  	<td class="resume-left"> IGCSE Examinations<br> June 2006 & 2007</td>
			  	<td class="resume-right">
				  	The IGCSE was administered in roughly 300 schools in India. I had the highest score in India in Extended Mathematics Without Coursework (June 2006; <a href="http://tisb.org/felicitation/press-releases/deccan-herald-igcse-students/">press release</a>), Physics (June 2007) and Geography (June 2007).
			 	</td>
			</tr>
		</table>
<!--
		<h2><a name="skills">Skills</a></h2>
		<table>
			<tr><center>(for some old quirky side-projects, see <a href="projects.html">Fun</a>)</center></tr>
			<tr>
			    <td class="resume-left"> Python</td>
			    <td class="resume-right">
			    	Primary language during my PhD. Personal emphasis on code reusability in contrast to the 'hacky' culture of bioinformatics. Commonly used libraries include numpy and scikit-learn.<br>
			    </td>
			</tr>
			<tr>
			    <td class="resume-left">Theano</td>
			    <td class="resume-right">
			    	Symbolic computation language used during PhD to implement custom features for deep learning projects.<br>
			    </td>
			</tr>
			<tr>
			  	<td class="resume-left"> Java</td>
			  	<td class="resume-right">Primary language at Palantir. In addition to being the primary developer for the healthcare team's first enterprise deployment, I wrote generalized software packages for use across different applications and teams. Used Spring for dependency injection.<br></td>
			</tr>
			<tr>
			    <td class="resume-left"> Perl</td>
			    <td class="resume-right">
			    	I used Perl for data cleanup and data integration at Palantir. It was also the primary language for my bioinformatics research at MIT. Chosen for ease of text processing and "quick and dirty" turnaround time. Of note, I built pipelines for data processing and QA.
			    </td>
			</tr>
			<tr>
			    <td class="resume-left"> SQL/PostgreSQL</td>
			    <td class="resume-right">SQL-like databases formed the original data store for many of Palantir's clients. Familiarity was necessary for import into Palantir's backends, and for writing data integrity checks.<br></td>
			</tr>
			<tr>
			    <td class="resume-left">R</td>
			    <td class="resume-right">
			    	No true bioinformaticist can avoid R for very long. I use R primarily for generating quick visualisations and for exploratory analysis of the data (eg: PCA).<br>
			    </td>
			</tr>
			<tr>
			    <td class="resume-left"> MATLAB</td>
			    <td class="resume-right">
			    	Several courses I have taken over the years have had homeworks/projects that were implemented in MATLAB. It was a particular favourite among Chemical Engineers (I began as a ChemE major before switching in my junior year).<br>
			    </td>
			</tr>
			<tr>
			    <td class="resume-left"> Javascript/CSS/HTML</td>
			    <td class="resume-right">
			    	I built prototypes for two web applications for deal proposals at Palantir, and built a web-based tool for the final project of a UI Design class at MIT.<br>
			    </td>
			</tr>
		</table>		
-->
		<h2><a name="selected-coursework">Selected Coursework</a></h2>
		<table>
			<tr>
			  <td class="a">Stanford</td>
			  <td class="b">Probabilistic Graphical Models</td>
			  <td class="c">CS 228</td>
			  <td class="d">Winter 2015</td>
			  <td class="e">A+</td>
			</tr>
			<tr>
			  <td class="a">Stanford</td>
			  <td class="b">Machine Learning</td>
			  <td class="c">CS 229</td>
			  <td class="d">Fall 2015</td>
			  <td class="e">A</td>
			</tr>
			<tr>
			  <td class="a">MIT</td>
			  <td class="b">Statistics for Applications</td>
			  <td class="c">18.443</td>
			  <td class="d">Spring 2013</td>
			  <td class="e">A</td>
			</tr>
			<tr>
			  <td class="a">MIT</td>
			  <td class="b">Advanced Computational Biology</td>
			  <td class="c">6.878</td>
			  <td class="d">Fall 2012</td>
			  <td class="e">A+</td>
			</tr>
			<tr>
			  <td class="a">MIT</td>
			  <td class="b">Design and Analysis of Algorithms</td>
			  <td class="c">6.046</td>
			  <td class="d">Spring 2012</td>
			  <td class="e">A</td>
			</tr>
			<tr>
			  <td class="a">MIT</td>
			  <td class="b">Software Construction</td>
			  <td class="c">6.005</td>
			  <td class="d">Spring 2012</td>
			  <td class="e">A+</td>
			</tr>
			<tr>
			  <td class="a">MIT</td>
			  <td class="b">Evolutionary Biology</td>
			  <td class="c">7.33</td>
			  <td class="d">Spring 2012</td>
			  <td class="e">A+</td>
			</tr>
		</table>		
	</div>
</body>
