<html>
<head>
	<title> Avanti Shrikumar</title>
	<link rel="stylesheet" href="default.css" type="text/css">
	<meta name="keywords" content="Avanti Shrikumar, Av Shrikumar">
</head>

<body>
	<link href="default.css" rel="stylesheet" type="text/css">	
	<div class="navigation" >
	<a class="menu" href="index.html"> Home </a>
	<a class = "present" href="resume.html"> Resume </a>
	<a class="menu" href="projects.html"> Fun </a>
	<a class="menu" href="bio.html"> Bio </a>
	
	<h1>
		<a class="present" href="resume.html"> Resume  </a>    
		
	</h1>
		
	</div>
	<div class="content">	
		<h3><a href="http://web.stanford.edu/~avanti/resume.pdf">(PDF)</a><h3>
		<h2> Education</h2>
		<table>
			<tr>
				<td class ="resume-left">Stanford
		  	<br> 2014 - <i>(2019)</i>
		    <br> Ph.D.
				  	</td> 
				<td class="resume-right"> 
					Ph.D. in Computer Science, applying deep learning to study regulatory genomics
				    <br>Ph.D. Advisor: Anshul Kundaje.<br>
				</td>
			</tr>
			<tr>
				<td class="resume-left"> MIT
				<br>2009 - 2013
				<br> B.S.
				</td>
				<td class="resume-right">
					B.S. Computer Science with Molecular Biology, Minor in Mathematics
					<br>Undergraduate GPA: 5.0 
				</td>
			</tr>
		</table>
		<h2> Experience </h2>
		<table>
			<tr>
		        <td class="resume-left">
		          Kundaje Lab
		          <br>
		          Sep 2014 - Present
		        </td>
		        <td class="resume-right">
					<ul><li>Conceived of and developed DeepLIFT, the first comprehensive interpretation engine for deep learning models in genomics.</li><li>Demonstrated the effectiveness of DeepLIFT on diverse genomic and epigenomic tasks, including motif discovery (where it achieves far superior performance relative to conventional methods).</li><li>Built frameworks to facilitate training deep learning models on genomic data.</li><li>Applied deep learning to predict genomic regulatory features from sequence.</li>
		        </td>
	      	</tr>
			<tr>
		        <td class="resume-left">
		          Palantir Technologies
		          <br>
		          June 2013 - Sep 2014
		        </td>
		        <td class="resume-right">
		         Forward Deployed Engineer for the Healthcare Team.<ul><li>Primary developer for pilot project which successfully became the Healthcare Team's first enterprise deployment. Continued as primary developer until I left the company.</li><li>Created a way to integrate Palantir's two major platforms into a seamless experience; success was highlighted company-wide.<li>Built multiple code libraries off-the-clock for use team-wide and company-wide.</li><li>3 ongoing patent filings/discussions.</li><li>Consistently rated high on bi-annual reviews, in which lead (Lekan Wang) told me I had at times done the work of 2-3 coders and had set a new standard for productivity among my teammates.</li></ul>
		        </td>
		      </tr>
		 	<tr>
		        <td class="resume-left">
		          MIT Kellis Lab
		          <br>
		          Sep 2012 - June 2013
		        </td>
		        <td class="resume-right">
		          Without supervision of a graduate student, applied a Hidden Markov Model to identify transcription factor binding sites using histone modification data. Won an award for outstanding research.
		        </td>
		    </tr>
	        <tr>
		        <td class="resume-left">
		          MIT Boyer Lab
		          <br>
		          Sep 2010 - Sep 2012
		        </td>
		        <td class="resume-right">
		          Analyzed high-throughput data on cardiac stem cells. Exercised autonomy on how best to process the data to answer biological questions. See publications. Supervisors: Professor Laurie Boyer, Dr. Joseph Wamstad.
		        </td>
	        </tr>
		</table>
		<h2>Talks</h2>
		<table>
			<tr>
				<td class="resume-left">
					CEHG Symposium <br> Feb 2016
				</td>
				<td class="resume-right">
					<em>Not Just a Black Box: Interpretable Deep Learning for Genomics and Epigenomics</em>. Abstract <a href=https://web.stanford.edu/dept/cehg/cgi-bin/cehg-symposium/speakers/avanti-shrikumar/>here</a>.
				</td>
			</tr>
			<tr>
				<td class="resume-left">
					Epigenomics<br> Feb 2016
				</td>
				<td class="resume-right">
					Similar content to CEHG 2016 talk.
				</td>
			</tr>
		</table>
		<h2>Publications</h2>
		<table>
			<tr>
				<td class="resume-left">
				Cell <br> Sep 2012
				</td>
				<td class="resume-right">
					Collaboration between Boyer lab at MIT and Gladstone Institutes. I performed the bulk of bioinformatics analysis at the Boyer lab. 204 citations as of Feb 2016.<br><br> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22981692">Dynamic and coordinated epigenetic regulation of developmental transitions in the cardiac lineage</a>. Cell 151, 206-220.<br><em>Wamstad JA, Alexander JM, Truty RM, <u>Shrikumar A</u>, Li F, Eilertson KE, Ding H, Wylie JN, Pico AR, Capra JA, Erwin G, Kattman SJ, Keller GM, Srivastava D, Levine SS, Pollard KS, Holloway AK, Boyer LA, Bruneau BG.</em><br>
				</td>
			</tr>
			<tr>
				<td class="resume-left">
				Circulation Research
				<br>Dec 2014
				</td>
				<td class="resume-right">
					Collaboration between Boyer lab at MIT and Lee lab at Harvard. I analysed RNA-seq data to study transcriptional reversion. <br><br><a href="http://www.ncbi.nlm.nih.gov/pubmed/25477501">Transcriptional Reversion of Cardiac Myocyte Fate During Mammalian Cardiac Regeneration.</a> Circ Res. 2014 Dec 4. pii: CIRCRESAHA.114.304269.<br><em>O'Meara CC, Wamstad JA, Gladstone RA, Fomovsky GM, Butty VL, <u>Shrikumar A</u>, Gannon JB, Boyer LA, Lee RT</em>
				</td>
			</tr>
		</table>
		<h2>Recognition</h2>
		<table>
			<tr>
			  <td class="resume-left">Microsoft Women's Fellowship<br>2016</td>
			  <td class="resume-right">Awarded to one woman per participating University pursuing or interested in pursuing a PhD</td>
			</tr>
			<tr>
			  <td class="resume-left"> Palantir company spotlight<br> Summer 2014</td>
			  <td class="resume-right">My primary deployment at Palantir, on which I was the main developer, was one of the few to leverage the strengths of the company's two major platforms, and Palantir repeatedly chose to highlight this success company wide.<br> 
			</tr>
			<tr>
			  <td class="resume-left"> Outstanding Research Award<br> Spring 2013</td>
			  <td class="resume-right">Awarded to select participants in the SuperUROP research program, for the project "Doubling the precision of DNA binding site prediction using a novel dataset" in the Kellis lab.<br> 
			</tr>
			<tr>
			  <td class="resume-left"> Sophomore Academic Excellence Award<br> Fall 2011</td>
			  <td class="resume-right">AIChE Sophomore Academic Excellence Award for the student with the highest GPA among chemical engineers after sophomore year at MIT.<br> 
			</tr>
			<tr>
			  	<td class="resume-left"> IGCSE Examinations<br> June 2006 & 2007</td>
			  	<td class="resume-right">
				  	The IGCSE was administered in roughly 300 schools in India. I had the highest score in India in Extended Mathematics Without Coursework (June 2006), Physics (June 2007) and Geography (June 2007).
				  	<br> 
			 	</td>
			</tr>
		</table>
		<h2>Skills</h2>
		<table>
			<tr><center>(for some old quirky side-projects, see <a href="projects.html">Fun</a>)</center></tr>
			<tr>
			    <td class="resume-left"> Python</td>
			    <td class="resume-right">
			    	Primary language during my PhD. Personal emphasis on code reusability in contrast to the 'hacky' culture of bioinformatics. Commonly used libraries include numpy and scikit-learn.<br>
			    </td>
			</tr>
			<tr>
			    <td class="resume-left">Theano</td>
			    <td class="resume-right">
			    	Symbolic computation language used during PhD to implement custom features for deep learning projects.<br>
			    </td>
			</tr>
			<tr>
			  	<td class="resume-left"> Java</td>
			  	<td class="resume-right">Primary language at Palantir. In addition to being the primary developer for the healthcare team's first enterprise deployment, I wrote generalized software packages for use across different applications and teams. Used Spring for dependency injection.<br></td>
			</tr>
			<tr>
			    <td class="resume-left"> Perl</td>
			    <td class="resume-right">
			    	I used Perl for data cleanup and data integration at Palantir. It was also the primary language for my bioinformatics research at MIT. Chosen for ease of text processing and "quick and dirty" turnaround time. Of note, I built pipelines for data processing and QA.
			    </td>
			</tr>
			<tr>
			    <td class="resume-left"> SQL/PostgreSQL</td>
			    <td class="resume-right">SQL-like databases formed the original data store for many of Palantir's clients. Familiarity was necessary for import into Palantir's backends, and for writing data integrity checks.<br></td>
			</tr>
			<tr>
			    <td class="resume-left">R</td>
			    <td class="resume-right">
			    	No true bioinformaticist can avoid R for very long. I use R primarily for generating quick visualisations and for exploratory analysis of the data (eg: PCA).<br>
			    </td>
			</tr>
			<tr>
			    <td class="resume-left"> MATLAB</td>
			    <td class="resume-right">
			    	Several courses I have taken over the years have had homeworks/projects that were implemented in MATLAB. It was a particular favourite among Chemical Engineers (I began as a ChemE major before switching in my junior year).<br>
			    </td>
			</tr>
			<tr>
			    <td class="resume-left"> Javascript/CSS/HTML</td>
			    <td class="resume-right">
			    	I built prototypes for two web applications for deal proposals at Palantir, and built a web-based tool for the final project of a UI Design class at MIT.<br>
			    </td>
			</tr>
		</table>		
		<h2>Selected Coursework</h2>
		<table>
			<tr>
			  <td class="a">Stanford</td>
			  <td class="b">Probabilistic Graphical Models</td>
			  <td class="c">CS 228</td>
			  <td class="d">Winter 2015</td>
			  <td class="e">A+</td>
			</tr>
			<tr>
			  <td class="a">Stanford</td>
			  <td class="b">Machine Learning</td>
			  <td class="c">CS 229</td>
			  <td class="d">Fall 2015</td>
			  <td class="e">A</td>
			</tr>
			<tr>
			  <td class="a">MIT</td>
			  <td class="b">Statistics for Applications</td>
			  <td class="c">18.443</td>
			  <td class="d">Spring 2013</td>
			  <td class="e">A</td>
			</tr>
			<tr>
			  <td class="a">MIT</td>
			  <td class="b">Advanced Computational Biology</td>
			  <td class="c">6.878</td>
			  <td class="d">Fall 2012</td>
			  <td class="e">A+</td>
			</tr>
			<tr>
			  <td class="a">MIT</td>
			  <td class="b">Design and Analysis of Algorithms</td>
			  <td class="c">6.046</td>
			  <td class="d">Spring 2012</td>
			  <td class="e">A</td>
			</tr>
			<tr>
			  <td class="a">MIT</td>
			  <td class="b">Software Construction</td>
			  <td class="c">6.005</td>
			  <td class="d">Spring 2012</td>
			  <td class="e">A+</td>
			</tr>
			<tr>
			  <td class="a">MIT</td>
			  <td class="b">Evolutionary Biology</td>
			  <td class="c">7.33</td>
			  <td class="d">Spring 2012</td>
			  <td class="e">A+</td>
			</tr>
		</table>		
	</div>
</body>
